src/extension/src/EnableCommandHandler.py

```diff
@@ -49,6 +49,11 @@ def execute_handler_action(self):
             self.ext_output_status_handler.read_file(self.seq_no)
 
             config_settings = self.ext_config_settings_handler.read_file(self.seq_no)
+
+            # set activity_id in telemetry
+            if self.logger.telemetry_writer is not None:
+                self.logger.telemetry_writer.set_operation_id(config_settings.__getattribute__(self.config_public_settings.activity_id))
+
```

Comment:
This is not best-effort...

Reply:
What should we do here?

Reply:
Doesn't need to be addressed in this PR if we're stopping execution as in the other comment early and just writing to the log file.